tox.ini

```diff
@@ -11,7 +11,11 @@ commands =
     coverage report -m
 
 [pytest]
-addopts = -l
+addopts = -l --mypy-same-process --mypy-ini-file=typesafety/mypy.ini
```

Comment:
that's quite awkward...`tox` for some reason chokes on that :/. It shouldn't, I know for sure it works because this is what I have been using for some time.